Libraries/Opc.Ua.Client/Subscription.cs

```diff
@@ -705,6 +778,8 @@ public void Create()
 
             AdjustCounts(ref revisedKeepAliveCount, ref revisedLifetimeCounter);
 
+            ManageMessageWorkerSemaphore();
```

Comment:
the new createasync may need this call too

Reply:
Ironically this call is functionally useless, it's in place in case the defaults ever need behavior to manage the worker semaphore. 

By default, workers are unlimited and sequential publishing is off. This means this method will leave the semaphore as null, as it started.

However, for avoiding accidental future issues (if for whatever reason, the default number of workers will be some number or sequential publishing will be the default behavior, which I don't think I recommend) then this will "protect" from that.

I'll add it to the new CreateAsync as well. Thanks for merging your recent Async library changes by the way, looked like a non-trivial merge.